.github/workflows/release.yml

```diff
@@ -93,6 +94,7 @@ jobs:
       matrix:
         os: ['macOS', 'Windows', 'Linux']
         python-version: ['3.5', '3.6', '3.7']
+      fail-fast: false
```

Comment:
Mostly LGTM, but wondering why we wouldn't want to fail fast on the pypi uploads? 

Failing fast doesn't  guarantee protection from uploading some but not all of the wheels;  though its possible it could stop it. I'm not seeing a benefit of preventing fast-fail on pypi upload?

Reply:
Oups my bad. I was mostly focused on testing the wheels in a fresh environement because I had a bug in this section earlier. Let me revert that.